hub/graphql/types/model_types.py

```diff
@@ -600,24 +608,17 @@ class Analytics:
     @strawberry_django.field
     def imported_data_count_by_region(self) -> List[GroupedDataCount]:
         data = self.imported_data_count_by_region()
-        # areas = models.Area.objects.filter(gss__in=[datum["gss"] for datum in data])
-        return [
-            GroupedDataCount(
-                **datum
-                # area_data=next((area for area in areas if area.gss == datum.get("gss", None)), None),
-            )
-            for datum in data
-        ]
+        return [GroupedDataCount(**datum) for datum in data]
```

Comment:
Do we not also need the area type here?

Reply:
we could add it, but it's not necessary for regions as there aren't duplicates

Reply:
Cool cool